### ISSUE TITLE
addon/plugin/costume component For Home Assistant

### ISSUE THREAD

**Issue by thirtythreeforty** [thirtythreeforty/neolink#45]
Migrated from #44:

> Hello All
> 
> It's possible to create a addon/plugin/costume component based in neolink to running in HA -Home Assistant
> 
> I think this can be useful once we can integrate the reolink cameras that is not supporting the RTSP or ONVIF protocols.
> 
> Converting in a supported devices to be available :)
> 
> Think about this please
> 
> Best regards

**Comment by thirtythreeforty** [comment]
@nunokaneco thanks for the request. I don't know much about Home Assistant. If you have a specific description about how it should be integrated, I'm open to facilitating, but I don't think I'll be able to personally work on HA integration.

**Comment by QuantumEntangledAndy** [comment]
I use homeassistant with neolink. Adding the following in the config as a genertic ffmpeg camera should work:

```
camera:
  - platform: ffmpeg
    name: Living Room Live
    input: -rtsp_transport tcp -probesize 500 -i rtsp://<NEOLINKIP>:8554/<NAME_CAM>
```

**Comment by Kaibob2** [comment]
Thanks for the effort, but i get "command not found" when running neolink --config my_config.toml in the Terminal on a Intel NUC on Ubuntu 20.04. My goal would be to integrate neolink into Homeassistant as well like @QuantumEntangledAndy did. What am I doing wrong?

**Comment by nunokaneco** [comment]
> h neolink. Adding the following in the config as a generic ffmpeg camera should work:

Ok thank you :) but I think if this is available as a addon it's more easy to install and work 

I'm try to put the neolink working, I also try to install from docker, also from direct program but without success for now ...

By the way are you use the neolink in same machine running the Home Assistant ? Can you please help me in order to put this working? 

Thank you

**Comment by Kaibob2** [comment]
@nunokaneco maybe follow here #51 . This seems quite promising.

**Comment by QuantumEntangledAndy** [comment]
Since @nunokaneco  got his issues fixed in #51 can we close this one?
